### PR TITLE
Add runEffCat

### PR DESCRIPTION
**Pull request by turion**

### CODE REVIEW

File: src/Control/Category/FreeEffect.hs [hunk 1]
```diff
@@ -89,6 +90,14 @@ foldNatEffCat
   -> c a b
 foldNatEffCat nat = foldNatFree2 (foldNatFree2 nat)
 
+-- | Join all effects in a free effectful category 'EffCat'.
```
Comment (coot) [inline src/Control/Category/FreeEffect.hs:93]:
```suggestion
-- | Join all effects in a free effectful category 'EffCat'.
--
```

**Review by coot — CHANGES_REQUESTED** [review]
LGTM, just a minor stylistic change.

**Review by coot — APPROVED** [review]
Thanks; please squash the two commits, and it can be merged.

**Comment by turion** [comment]
Done.

**Pull request merged**